.github/workflows/nightly.yaml

```diff
@@ -1,4 +1,5 @@
-name: Nightly Build for rolling
+# This is nightly workflow for each distribution
+name: Nightly Build for distribution
```

Comment:
this nightly build yaml is meant to be used for any downstream branches, just change the target ros2 distro name.